packages/icons-ui/package.json

```diff
@@ -55,8 +55,6 @@
         "@spectrum-web-components/iconset": "1.5.0"
     },
     "devDependencies": {
-        "@spectrum-css/ui-icons": "^1.1.2",
```

Comment:
The ui-icons package only contains SVG assets - I think that's out of scope for this work?